### PR TITLE
Delete event API method

### LINKED ISSUE #65: Set up DELETE endpoint for remove event

### PR DESCRIPTION
**Pull request by Lucy-Zh**
Closes #65

### CODE REVIEW

File: web/src/components/Schedule/Schedule.jsx [hunk 1]
```diff
@@ -47,6 +47,14 @@ export default function Schedule() {
     getAllEvents();
   }
 
+  async function deleteEvent(id) {
```
Comment (samperezh) [inline web/src/components/Schedule/Schedule.jsx:50]: I would avoid including this file in the pull request as the frontend in main is entirely different now. I personally would make the backend-frontend connection in a later PR.

**Review by samperezh — CHANGES_REQUESTED** [review]
Address comment left on Schedule.jsx before merging

File: web/src/components/Schedule/Schedule.jsx (hunk 1 above)
Comment (Lucy-Zh) [reply web/src/components/Schedule/Schedule.jsx]: makes sense, just removed the front-end in the latest commit!!

**Review by samperezh — APPROVED** [review]
LGTM!

**Pull request merged**